com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -874,12 +971,19 @@ public bool supportsSoftShadows
             internal set { m_SoftShadowsSupported = value; }
         }
 
+        /// <summary>
+        /// Specifies if this <c>UniversalRenderPipelineAsset</c> should use dynamic batching.
+        /// </summary>
+        /// <see href="https://docs.unity3d.com/Manual/DrawCallBatching.html"/>
         public bool supportsDynamicBatching
         {
             get { return m_SupportsDynamicBatching; }
             set { m_SupportsDynamicBatching = value; }
         }
 
+        /// <summary>
+        /// Returns true if the Render Pipeline Asset supports mixed lighting, false otherwise.
```

Comment:
suggestion add reference to Mixed Lighting docs page.

Reply:
I agree